app/models/katello/glue/candlepin/owner.rb

```diff
@@ -39,14 +39,6 @@ def content_access_mode
         self.owner_details['contentAccessMode']
       end
 
-      def pools(consumer_uuid = nil)
-        if consumer_uuid
-          Resources::Candlepin::Owner.pools self.label, :consumer => consumer_uuid
-        else
-          Resources::Candlepin::Owner.pools self.label
-        end
-      end
```

Comment:
This method was masking the 'pools' relation.  It seems like its not used anywhere.